esp/esp/survey/migrations/0002_auto_20200301_1048.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def create_question_types(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    QuestionType = apps.get_model("survey", "QuestionType")
+    QuestionType.objects.using(db_alias).get_or_create(name="Instruction Text", _param_names = "Text")
+    QuestionType.objects.using(db_alias).get_or_create(name="Long Answer", _param_names = "Rows")
```

Comment:
Even though I applied the migrations and refreshed the cache, I still can't get the columns parameter to go away :(

Reply:
Yeah, the migration isn't really designed to _fix_ the QuestionTypes, just to set them up in the first place.